components/mgmtworker/scripts/start.py

```diff
@@ -10,7 +11,30 @@
 import utils  # NOQA
 
 MGMT_WORKER_SERVICE_NAME = 'mgmtworker'
+CELERY_PATH = '/opt/mgmtworker/env/bin/celery'  # also hardcoded in create
+
+ctx_properties = utils.ctx_factory.get_install_properties(
+    MGMT_WORKER_SERVICE_NAME)
+
+
+@utils.retry(subprocess.CalledProcessError)
+def check_worker_running(amqp_url):
+    """Use `celery status` to check if the worker is running."""
+    return utils.sudo([
```

Comment:
you seem to be returning here even though you're not doing anything with the return value..